### ISSUE TITLE
Error while generating

### ISSUE THREAD

**Issue by hberndt** [jhipster/jhipster-kotlin#27]
I setup as mentioned in README. And started generating a new app:
```jhipster --blueprint generator-jhipster-kotlin```

I get the following error:
```
Error: ERROR! Copying template ../../../node_modules/generator-jhipster/generators/server/templates/src/main/java/package/config/ThymeleafConfiguration.java.ejs failed. 
[Error: ENOENT: no such file or directory, open '/Users/hberndt/Projects/jhipster-kotlin/node_modules/generator-jhipster/generators/server/templates/src/main/java/package/config/ThymeleafConfiguration.java.ejs']
```

I am very interested in contributing to this project because I like kotlin and made a kotlin microservice based on prior JHipster generated app manuelly.

Any help appreciated.

JHipster Version(s): actual master branch (linked via yarn link)

[X] Checking this box is mandatory (this is just to show you read everything)

**Comment by sendilkumarn** [comment]
This is because `ThymeleafConfiguration.java` file is removed from the main generator. I will do a patch release today. Sorry for inconvenience. 

Sure, you are welcome to contribute. 🎉

**Comment by sendilkumarn** [comment]
Closed by 6b1068453206fdd14b6bdc8d760b3709bc7884c5

Will be available in version `0.2.0`
